shared/metrics/__init__.py

```diff
@@ -9,14 +9,28 @@
 
 __all__ = [
     "Counter",
+    "Gauge",
     "Histogram",
     "Summary",
     "start_prometheus",
 ]
 
 
-def inc_counter(counter: Counter, labels: dict) -> None:
+def inc_counter(counter: Counter, labels: dict = None) -> None:
```

Comment:
```suggestion
def inc_counter(counter: Counter, labels: dict | None = None) -> None:
```

Same below. The default value has to match the correct type.

Reply:
good catch, thanks!